src/main/java/slimeknights/tconstruct/blocks/CommonBlocks.java

```diff
@@ -0,0 +1,125 @@
+package slimeknights.tconstruct.blocks;
+
+import net.minecraft.block.Block;
+import net.minecraft.item.Item;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.registries.ObjectHolder;
+
+import slimeknights.mantle.block.StairsBaseBlock;
+import slimeknights.tconstruct.TConstruct;
+import slimeknights.tconstruct.common.registry.BaseRegistryAdapter;
+import slimeknights.tconstruct.common.registry.ItemRegistryAdapter;
+import slimeknights.tconstruct.library.TinkerRegistry;
+import slimeknights.tconstruct.shared.block.ConsecratedSoilBlock;
+import slimeknights.tconstruct.shared.block.FirewoodBlock;
+import slimeknights.tconstruct.shared.block.FirewoodSlabBlock;
+import slimeknights.tconstruct.shared.block.GlowBlock;
+import slimeknights.tconstruct.shared.block.GraveyardSoilBlock;
+import slimeknights.tconstruct.shared.block.GroutBlock;
+import slimeknights.tconstruct.shared.block.MetalBlock;
+import slimeknights.tconstruct.shared.block.SlimyMudBlock;
+
+import static slimeknights.tconstruct.common.TinkerPulse.injected;
+
+/** General things and crafting related blocks */
+@SuppressWarnings("unused")
+@ObjectHolder(TConstruct.modID)
+public class CommonBlocks {
+
+  /* Crafting related items */
+  public static final GroutBlock grout = injected();
+  public static final GraveyardSoilBlock graveyard_soil = injected();
+  public static final ConsecratedSoilBlock consecrated_soil = injected();
+  public static final SlimyMudBlock slimy_mud_green = injected();
+  public static final SlimyMudBlock slimy_mud_blue = injected();
+  public static final SlimyMudBlock slimy_mud_magma = injected();
+  public static final FirewoodBlock lavawood = injected();
+  public static final FirewoodBlock firewood = injected();
```

Comment:
These are only used by the tools module, I wonder if it would be worth moving them to ToolsBlocks once we port that module over.

Likewise grout is only used by the smeltery.

Reply:
Same as with the slimeblocks